CHANGELOG.md

```diff
@@ -9,6 +9,9 @@
 #### Upgraded Ruby interpreters:
 * Add support for Ruby 2.2.8, 2.3.5 and 2.4.2 [\#4159](https://github.com/rvm/rvm/pull/4159)
 
+#### Binaries:
+* Ubuntu 16.04 x64 binaries for Ruby 2.2.8, 2.3.5 and 2.4.2 [\#4161](https://github.com/rvm/rvm/issues/4161)
```

Comment:
It looks like another PR snuck in front, so this is #4162.

Reply:
Yes, but the issue is #4161, so all good :)

Reply:
Ah, nice - nevermind!